code/frameworks/react-vite/src/types.ts

```diff
@@ -25,6 +29,19 @@ type StorybookConfigFramework = {
   };
 };
 
+type TypescriptOptions = TypescriptOptionsBase & {
+  /**
+   * Sets the type of Docgen when working with React and TypeScript
+   *
+   * @default `'react-docgen-typescript'`
+   */
+  reactDocgen: 'react-docgen-typescript' | 'react-docgen' | false;
+  /**
+   * Configures `@joshwooding/vite-plugin-react-docgen-typescript`
+   */
+  reactDocgenTypescriptOptions: Parameters<typeof docgenTypescript>;
```

Comment:
Should this be `Parameters<typeof docgenTypescript>[0]`? I think parameters returns an array